### PR TITLE
mini-sftp-server: Add version 2.25

### PR DESCRIPTION
**Pull request by mark05e**
- Closes https://github.com/lukesampson/scoop-extras/issues/4375

### CODE REVIEW

File: bucket/mini-sftp-server.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "version": "2.25",
+    "description": "A simple mini sftp server with GUI controls",
+    "homepage": "http://www.coreftp.com/server/",
+    "license": "Freeware",
```
Comment (Ash258) [inline bucket/mini-sftp-server.json:5]: Source?

Comment (mark05e) [reply bucket/mini-sftp-server.json:5]: > Source?

Are you asking about the source for the license details? If yes - it is mentioned on the same page.

image

If you are asking about the source code - there is none available.

**Comment by mark05e** [comment]
> Where are configuration stored?

As per this [discussion on portablefreeware.com](https://www.portablefreeware.com/forums/viewtopic.php?p=29438#p29438), it seems to be writing in the registry. 😓

**Pull request merged**